javascript-modal/main.js

```diff
@@ -0,0 +1,16 @@
+var $openModalButton = document.querySelector('.open-modal-button');
+
+var $modal = document.querySelector('.modal');
+
+var $noButton = document.querySelector('.no-button');
+
+function modalClick(event) {
+  $modal.style.display = 'block';
+}
+
+function noButtonClick(event) {
+  $modal.style.display = 'none';
+}
```

Comment:
This is inline styling, which is _really_ bad news. Instead, you should make a class that you can either add or remove to hide/show the element.